### PR TITLE
[bepis] Factorio 14.23 compatibility updates

### PR DESCRIPTION
**Pull request by philihp**
* Added steps necessary for someone to start up a node on AWS, because nobody wanna run a VM locally.
* Updates to Factorio for newer version, removed old params. Updated params that want .zip on them.
* Added myself.

@justjake @sjzhu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,47 +43,21 @@ project. I did `rbenv install 2.2.3` and then `rbenv local 2.2.3` to use ruby
...
+1. `vi nodes/12.34.56.78.json` and add roles to the run_list. These are the recipes that your box
+   will run, e.g.
+   `["recipe[base]","recipe[factorio]","recipe[factorio::api","recipe[chopped_nginx"]`.
+1. `bundle exec knife solo cook ubuntu@12.34.56.78 -i ~/.ssh/leroy_jenkins.pem` and chef will go off
+   and run all of the cheffy stuff.
 
 On future runs you should be using your own user account to cook and clean the
 box, since the vagrant user may no longer be allowed to SSH.
-
-### Testing chef in Vagrant
```
Comment (justjake) [inline README.md]: I like the vagrant stuff because it lets you provision a new system to test in like 30 secs.

**Review by justjake — CHANGES_REQUESTED** [review]
Seems legit.

Please move the Vagrant stuff to the bottom of the README and retain it! I like vagrant, and it's useful for iterating quickly.

Chef changes good to go.

**Comment by philihp** [comment]
@justjake PTAL

**Review by justjake — APPROVED** [review]

**Comment by justjake** [comment]
@philihp LGTM

**Pull request merged**